src/c2s/mongoose_c2s.erl

```diff
@@ -534,6 +568,12 @@ handle_info(StateData, _C2SState, replaced) ->
     send_element_from_server_jid(StateData, StreamConflict),
     send_trailer(StateData),
     {stop, {shutdown, replaced}};
+handle_info(_StateData, _C2SState, {store_session_info, Jid, Key, Value, _FromPid}) ->
```

Comment:
Because in logs I saw this error somewhere, but it probably wasn't because of this sm_suite exactly, probably in some other suite. Will remove from here and add it when any other test actually fails for this.